tests/substrate.rs

```diff
@@ -241,18 +245,21 @@ impl Externals for MockSubstrate {
                 println!("seal_clear_storage: {:?}", key);
                 self.store.remove(&(self.vm.account, key));
 
-                Ok(None)
+                // The size of the pre-existing value is returned (if any).
+                // In this mock VM we just always assume that there was no value and return `None`.
+                Ok(Some(NONE_SENTINEL))
```

Comment:
Rather than returning a bogus value, you could just do something like:
```rust
let old_length = self.store.remove(&(self.vm.account, key)).map(|e| e.len()).unwrap_or(0);
return Ok(Some(Runtime:i32(old_length as i32));
```

Reply:
ACK, but would that then be `unwrap_or(SENTINEL_VALUE)`?

Reply:
Well, whatever substrate returns if there was no previouskey